appshell/appshell_extensions_gtk.cpp

```diff
@@ -164,15 +164,21 @@ int32 ShowOpenDialog(bool allowMultipleSelection,
                      CefRefPtr<CefListValue>& selectedFiles)
 {
     GtkWidget *dialog;
-    const char* dialog_title = chooseDirectory ? "Open Directory" : "Open File";
+    // const char* dialog_title = chooseDirectory ? "Open Directory" : "Open File";
```

Comment:
I think we can get rid of this line, since the caller provides the specific title.